site/docs/components/country-symbol/index.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 title: Country symbol
 data:
-  description: 'Country symbols display an icon that represents the flag of a country or intergovernmental organization. You can use country symbols individually or as a complete package. The country symbols are stylized and representative, so users are more likely to recognize them at smaller sizes, and they maintain their character within the circular form. Rectangular "sharp" versions are also provided for every country symbol.'
+  description: 'Country symbols display an icon that represents the flag of a country or territory. You can use country symbols individually or as a complete package. The country symbols are abstractly stylized, so users are more likely to recognize them at smaller sizes. Though we maintain their character within the circular form, we also provide rectangular "sharp" versions are for every country symbol.'
```

Comment:
"we also provide rectangular "sharp" versions **are** for every country symbol."